contracts/rAura.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import { ERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/ERC20.sol";
+import { Ownable } from "@openzeppelin/contracts-0.8/access/Ownable.sol";
+
+/**
+ * @title   rAura
+ */
+contract rAura is ERC20, Ownable {
+    address public operator;
+
+    /* ========== EVENTS ========== */
```

Comment:
I would like to see more of these throughout.. 

"LAYER ZERO"

"CALLED FROM BOOSTER"

"ADMIN"

"USER"

Reply:
Updated the comments on SiphonDepositor and SiphonReceiver which are the ones with all the action. Removed the extra bits in this contract don't know where operator and that event came from, bad copy pasta 😂